nautobot/docs/development/getting-started.md

```diff
@@ -648,6 +648,27 @@ To enforce best practices around consistent [coding style](style-guide.md), Naut
 | `invoke flake8`         | `flake8`                     |
 | `invoke black`          | `black`                      |
 
+## Working on Documentation
+
+Some features require documentation updates or new documentation to be written. The documentation files can be found in the `docs` directory. To preview these changes locally, you can use `mkdocs`.
+
+### Installing `mkdocs`
+
+If you are using the poetry-based workflow, `mkdocs` should already be installed in your environment. This section mostly applies if you are using Docker to manage your development environment.
+
+The `mkdocs` command can be installed via pip, either globally or in your virtual environment.
+
+```
+pip install mkdocs mkdocs mkdocs-include-markdown-plugin
```

Comment:
In other examples in this document, we're explicitly using `pip3` rather than `pip` - might be good to be consistent.
```suggestion
$ pip3 install mkdocs mkdocs-include-markdown-plugin
```